include/rocksdb/db.h

```diff
@@ -1135,20 +1135,16 @@ class DB {
   // Simpler versions of the GetApproximateSizes() method above.
   // The include_flags argument must of type DB::SizeApproximationFlags
   // and can not be NONE.
-  virtual Status GetApproximateSizes(ColumnFamilyHandle* column_family,
-                                     const Range* ranges, int n,
-                                     uint64_t* sizes,
-                                     uint8_t include_flags = INCLUDE_FILES) {
-    SizeApproximationOptions options;
-    options.include_memtables =
-        (include_flags & SizeApproximationFlags::INCLUDE_MEMTABLES) != 0;
-    options.include_files =
-        (include_flags & SizeApproximationFlags::INCLUDE_FILES) != 0;
-    return GetApproximateSizes(options, column_family, ranges, n, sizes);
-  }
-  virtual Status GetApproximateSizes(const Range* ranges, int n,
-                                     uint64_t* sizes,
-                                     uint8_t include_flags = INCLUDE_FILES) {
+  virtual Status GetApproximateSizes(
+      ColumnFamilyHandle* column_family, const Range* ranges, int n,
+      uint64_t* sizes,
+      uint8_t include_flags =
```

Comment:
iiuc, our goal is to change `uint8_t` to `SizeApproximateFlags` which is an enum class so that previous code that calls the `GetApproximateSizes(..., bool)` will fail to compile.

Reply:
Oh ok. I thought other way round. I will update that.